mcs/tools/gacutil/gacutil.exe.sources

```diff
@@ -1,2 +1,9 @@
 driver.cs
 ../security/StrongNameManager.cs
+../../../external/ikvm/reflect/*.cs
+../../../external/ikvm/reflect/Emit/*.cs
+../../../external/ikvm/reflect/Metadata/*.cs
```

Comment:
That's a good point. I'll try to trim it a bit. (Just removing the `Emit/*.cs` line entirely didn't work, so I didn't try digging into it more yet)